azure-pipelines.yml

```diff
@@ -193,16 +193,8 @@ jobs:
       - task: CmdLine@2
         displayName: 'Run Unit Tests'
         inputs:
-          script: 'dotnet test -c Release $(PathToCommunityToolkitUnitTestCsproj) --settings ".runsettings" --collect "XPlat code coverage" --logger trx --results-directory $(Agent.TempDirectory)'
+          script: 'dotnet run -c Release --project $(PathToCommunityToolkitUnitTestCsproj) --results-directory $(Agent.TempDirectory) --coverage --coverage-output $(Agent.TempDirectory)/coverage.cobertura.xml --coverage-output-format cobertura'
 
-      - task: PublishTestResults@2
```

Comment:
Maybe you can include code coverage task as well

Reply:
With Azure Pipelines, it is already included 

<img width="1496" alt="image" src="https://github.com/user-attachments/assets/3aac76ef-ea66-4bbd-9f01-841b58b8e16b" />

Reply:
I missed we are using azure pipelines here.